packages/rocket-uploads3-store-event-aws-infrastructure/src/s3-notification-event-store.ts

```diff
@@ -0,0 +1,68 @@
+import { Duration, RemovalPolicy, Stack } from '@aws-cdk/core'
+import { Bucket, EventType } from '@aws-cdk/aws-s3'
+import { RocketUtils } from '@boostercloud/framework-provider-aws-infrastructure'
+import { Function, Runtime, Code } from '@aws-cdk/aws-lambda'
+import { Table } from '@aws-cdk/aws-dynamodb'
+import { S3EventSource } from '@aws-cdk/aws-lambda-event-sources'
+import { createPolicyStatement } from '@boostercloud/framework-provider-aws-infrastructure/dist/infrastructure/stacks/policies'
+
+const path = require('path')
+
+export type AWSS3EventsLambdaParams = {
+  bucketName: string
+  eventTypeName: string
+  entityTypeName: string
+}
+
+export class S3NotificationEventStoreStack {
+  public static mountStack(params: AWSS3EventsLambdaParams, stack: Stack): void {
+    const appName = stack.stackName.split('-app')[0]
+
+    // Create new bucket to upload file
+    const sourceBucket = new Bucket(stack, 'sourceUploadBucket' + params.entityTypeName, {
+      bucketName: params.bucketName,
+      removalPolicy: RemovalPolicy.DESTROY,
+    })
+
+    const eventsStore = stack.node.tryFindChild('events-store') as Table
```

Comment:
Why not pass the events-store table as a parameter? Or `events-store` is fixed and isn't configurable? Makes sense?

Reply:
Since you have the names of the resources inside the Booster config  object you could use that one passing by via rocket parameters:

```typescript
Booster.configure('production', (config: BoosterConfig): void => {
  config.appName = 'process-files'
  config.provider = Provider([
    {
      packageName: '@boostercloud/rocket-uploads3-store-event-aws-infrastructure',
      parameters: {
        bucketName: 'process-big-file-rocket',
        eventTypeName: 'BigFileAdded',
        entityTypeName: 'BigFile',
        eventStore: config.resourceNames.eventsStore,
      },
    }
  ])
})
```
And then in do this `const eventsStore = stack.node.tryFindChild(params.eventStore) as Table`

Reply:
@charlietfe This was my initial implementation but I changed to the current one. I think exposing the event store name to the end user isn't a good idea. 

The main reason behind this reasoning is, this event store name can only have one valid value and only one. By exposing this as a param to the users, we are giving them the chance to make a mistake and put a different value meaning the rocket will fail. I really believe we shouldn't expose this value to them.

Also, I believe the event store name is an internal detail of the framework.

I would keep it as it is, thoughts? @javiertoledo @charlietfe @NickSeagull ?

Reply:
Agree with @rdoria1

Reply:
@rdoria1 Now that the `config` object is passed to all rockets (the PR was merged), you can get now the name from it 😃